From 2c3896878ba8118b48c55528c6bac98f05f223f3 Mon Sep 17 00:00:00 2001
From: Mikko Rapeli <mikko.rapeli@linaro.org>
Date: Thu, 3 Jul 2025 13:34:21 +0000
Subject: [PATCH] build-certs.py: use /usr/bin/env for python3

Avoids errors if python3 install path is not
in /usr/bin.

Signed-off-by: Mikko Rapeli <mikko.rapeli@linaro.org>
---
 data/tests/build-certs.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Upstream-Status: Submitted [https://github.com/fwupd/fwupd/pull/8998]

diff --git a/data/tests/build-certs.py b/data/tests/build-certs.py
index 3b8503b72bcf..222e8a4a0f8b 100755
--- a/data/tests/build-certs.py
+++ b/data/tests/build-certs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 # SPDX-License-Identifier: LGPL-2.1+
 
 import os
-- 
2.43.0

